edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1449,7 +1449,8 @@ export default {
           set(rkeConfig.chartValues, name, userValues);
         }
       });
-    }
+    },
+    get
```

Comment:
The biggest nitpick in the world, but I would probably prefer `get` grouped with similar methods at the top, along with `nlToBr` & `set`.